src/bindings/js/node/lib/index.ts

```diff
@@ -0,0 +1,20 @@
+const ov = require('../build/Release/ov_node_addon.node');
```

Comment:
```suggestion
const ov: ovNodeModule = require('../build/Release/ov_node_addon.node');
```

Reply:
I also had to import ovNodeModule:
`import loadModel, { ovNodeModule } from './node-model';`